pom.xml

```diff
@@ -132,7 +132,7 @@ flexible messaging model and an intuitive client API.</description>
     <rocksdb.version>6.29.4.1</rocksdb.version>
     <slf4j.version>1.7.32</slf4j.version>
     <commons.collections4.version>4.4</commons.collections4.version>
-    <log4j2.version>2.17.1</log4j2.version>
+    <log4j2.version>2.18.0</log4j2.version>
```

Comment:
We also need to update versions in license files.

Reply:
OK